src/sounds.cpp

```diff
@@ -272,14 +272,13 @@ void sounds::process_sound_markers( player *p )
             p->add_effect( effect_deaf, deafness_duration );
             if( p->is_deaf() && !is_deaf ) {
                 is_deaf = true;
-                sfx::do_hearing_loss( to_turns<int>( deafness_duration ) );
```

Comment:
because this is how game.cpp looks:
```
    sounds::process_sound_markers( &u );

    if( u.is_deaf() ) {
        sfx::do_hearing_loss();
    }
```